### PR TITLE
[Ukrainian] Translate `/resources/about.md`

### PR DESCRIPTION
**Pull request by deptyped**

### CODE REVIEW

File: site/docs/uk/resources/about.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Про grammY
+
+## Що таке grammY?
+
+grammY - це програмне забезпечення, завдяки якому ви можете запрограмувати свій власний [чат бот](https://core.telegram.org/bots) для [Telegram Messenger](https://telegram.org).
```
Comment (LWJerri) [inline site/docs/uk/resources/about.md:5]: Probably correct translating is: "...свого власного [чат-бота](https://core.telegram.org/bots)..."

File: site/docs/uk/resources/about.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Коли був створений grammY?
+
+Вперше код grammY був опублікований наприкінці березня 2021 року.
+Через кілька тижнів він набув першої стабільної версії.
+
+## Хто розробляє grammY?
+
+grammY розробляється волонтерами, які працюють над бібліотекою, документацією та рештою екосистеми у свій вільний час.
+Ви можете знайти список усіх учасників в [README](https://github.com/grammyjs/grammY#contributors-) нашого репозиторію.
```
Comment (LWJerri) [inline site/docs/uk/resources/about.md:17]: Maybe change "нашого" to "головного"? After all, in the link, we are referring to the main repository of the project.

File: site/docs/uk/resources/about.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Ви можете зв'язатися з нами у [англомовному чаті](https://t.me/grammyjs)!
+Якщо ви розмовляєте російською, ви також можете приєднатися до [російськомовного чату](https://t.me/grammyjs_ru)!
+
+## Як розробляється grammY?
+
+grammY є повністю безкоштовним програмним забезпеченням з відкритим кодом.
+Код доступний в репозиторії на [GitHub](https://github.com/grammyjs/grammY).
+
+Ми вітаємо участь усіх бажаючих.
```
Comment (LWJerri) [inline site/docs/uk/resources/about.md:27]: On my mind, better translation also can be: "Ми вітаємо участь у розвитку проєкту всіх бажаючих."

File: site/docs/uk/resources/about.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+Всі зміни коду перевіряються кількома учасниками, поетапно оптимізуються та ретельно тестуються, зокрема на реальних проєктах.
+
+## Яка мова програмування використовується в grammY?
+
+grammY написаний повністю на мові TypeScript, яка є надмножиною мови JavaScript.
+Отже, grammY працює на Node.js.
+
+Також grammY може працювати на Deno, який позиціонує себе як наступник Node.js.
+(Технічно можна запустити grammY навіть у сучасних браузерах, хоча це вкрай рідко може знадобитися.)
```
Comment (LWJerri) [inline site/docs/uk/resources/about.md:36]: The dot in parentheses is unnecessary, in my opinion.

File: site/docs/uk/resources/about.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Якщо ви прийшли з іншої мови програмування або фреймворку, ви можете ознайомитися з [детальним порівнянням фреймворків](./comparison.md).
+
+## Як можна допомогти проєкту?
+
+Існує кілька способів підтримати цей проєкт!
+
+- Надішліть [pull request](https://github.com/grammyjs/grammY/pulls) зі змінами коду.
+- [Повідомте](https://github.com/grammyjs/grammY/issues/new) про виявлений баг, бажану функцію чи щось інше.
+- Покращіть [документацію](https://github.com/grammyjs/website).
+- Допомагайте користувачам у [англомовному](https://t.me/grammyjs) чи [російськомовному](https://t.me/grammyjs_ru) чатах спільноти.
```
Comment (LWJerri) [inline site/docs/uk/resources/about.md:49]: Also, we can translate this text as "Допомагайте користувачам у [англомовних](https://t.me/grammyjs) або [російськомовних](https://t.me/grammyjs_ru) чатах."

**Review by LWJerri — CHANGES_REQUESTED** [review]

File: site/docs/uk/resources/about.md (hunk 5 above)
Comment (deptyped) [reply site/docs/uk/resources/about.md:49]: Not sure about that. If we're talking about one chat for each language, why use the plural forms "англомовних чатах" and "російськомовних чатах"?

File: site/docs/uk/resources/about.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+Ми вітаємо участь у розвитку проєкту всіх бажаючих.
+Всі зміни коду перевіряються кількома учасниками, поетапно оптимізуються та ретельно тестуються, зокрема на реальних проєктах.
+
+## Яка мова програмування використовується в grammY?
+
+grammY написаний повністю на мові TypeScript, яка є надмножиною мови JavaScript.
+Отже, grammY працює на Node.js.
+
+Також grammY може працювати на Deno, який позиціонує себе як наступник Node.js.
+(Технічно можна запустити grammY навіть у сучасних браузерах, хоча це вкрай рідко може знадобитися)
```
Comment (Nazar-Ant) [inline site/docs/uk/resources/about.md:36]:
```suggestion
Технічно можна запустити grammY навіть у сучасних браузерах, хоча це вкрай рідко може знадобитися.
```

**Review by Nazar-Ant — CHANGES_REQUESTED** [review]
Finally, let's resolve the changes, and then I'll accept those changes

**Review by Nazar-Ant — APPROVED** [review]

**Review by Nazar-Ant — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]
Complete review and this PR ready to merge.

**Pull request merged**